pkgs/kubeshark/kubeshark/registry.yaml

```diff
@@ -0,0 +1,21 @@
+packages:
+  - type: github_release
+    repo_owner: kubeshark
+    repo_name: kubeshark
+    asset: kubeshark_{{.OS}}_{{.Arch}}
+    description: The API traffic viewer for Kubernetes providing real-time, protocol-aware visibility into Kubernetes internal network capturing, dissecting and monitoring all traffic and payloads going in, out and across containers, pods, nodes and clusters.
+    supported_envs:
+      - windows/amd64
+      - darwin
+      - linux
+    overrides:
```

Comment:
Please add `format: raw` since the asset is not an archive.

Reply:
kubeshark releases both `raw` and `tar.gz`. https://github.com/kubeshark/kubeshark/releases/tag/38.5
I prefer `tar.gz` than `raw` because the file size is small.